buildpack.toml

```diff
@@ -32,5 +32,6 @@ include-files = [
   "bin/build",
   "bin/detect",
   "bin/main",
+  "bin/execd_wraper.sh",
```

Comment:
```suggestion
  "bin/execd_wrapper.sh",
```

Reply:
Not sure if this change was intentional, but it looks like a typo. If you change this, you also do not need to modify the scripts/build.sh file.

```
> create-package --destination ./out --version foo --source .

Profile Buildpack foo
  https://github.com/buildpacks/profile
  Creating package in ./out
  Pre-package with scripts/build.sh
    Adding LICENSE
    Adding NOTICE
    Adding README.md
    Adding bin/build
    Adding bin/detect
    Adding bin/execd_wrapper.sh
    Adding bin/main
    Adding buildpack.toml
```

The `create-package` tool will automatically include that file in your buildpack.